serveradmin/serverdb/models.py

```diff
@@ -126,6 +127,9 @@ def is_unique_ip(
     # only withing the same attribute id. That means different hosts can have
     # the same IP address as long as it is in different attributes.
 
+    duplicates = []
+    object_id = server.server_id
```

Comment:
Why passing the whole server object when we only needing the object_id ?

Reply:
This commit also improves error message where `server.hostname` is used, which greatly helps when committing multiple objects. Hence changing the attribute from `object_id` to `server`.

Reply:
Okay fair enough. 

Just be aware that with that you force the method to be aware of the structure of the Server model implicitly and while it is currently the case we might not always keep hostname as unique value but only object_id.